### ISSUE TITLE
Define XXX (strings/Functions etc.)

### ISSUE THREAD

**Issue by Maijin** [hteso/iaito#91]
All of those define capability should be available by right click:

![image](https://cloud.githubusercontent.com/assets/1408600/24706722/01f434d0-1a11-11e7-8bbf-c6ee5104fada.png)

**Comment by hteso** [comment]
Agreed. There are lots of options there, so I'll start now and I should have a bunch of them for the 1.0 release, but I don't expect to have them all until the 1.1.
